fixes.inc

```diff
@@ -205,6 +205,15 @@
  * AUTHOR:   im
  * AUTHOR:   Y_Less (https://github.com/Y-Less/)
  *
+ * FIX:      OnPlayerEnterVehicle_3
+ * PROBLEM:  No player animation when trying to enter the driver door of a locked vehicle
+ * SOLUTION: Leave the door unlocked and activate an animation when the player attemps
+ *           to enter a 'locked' vehicle.
+ * SEE:      "OnPlayerEnterVehicle".
+ * AUTHOR:   ziggi (https://github.com/ziggi)
+ * AUTHOR:   rt-2 (https://github.com/Y-Less/)
+ * POST:     https://github.com/ziggi/samp-enterfix
```

Comment:
I wish I was Y_Less, the equivalent of Jesus to the SAMP Forum.
Anyhow I have still been uncovered: I copy my stuff from Y_Less works haha.